qemu/tests/cfg/nvdimm_mapsync.cfg

```diff
@@ -0,0 +1,28 @@
+- nvdimm_mapsync:
```

Comment:
This case have many almost same cfg with common part nvdimm.cfg. Prefer to use same cfg but different script.

Reply:
Updated, please help review again, thanks.